.travis.yml

```diff
@@ -7,7 +7,11 @@ php:
     - 7.1
 
 before_script:
-  - travis_retry composer update --no-interaction
+    - travis_retry composer update --no-interaction
 
 script:
-    ./vendor/bin/phpunit --testsuite tests
+    - vendor/bin/phpcs -p --warning-severity=0 src/ tests/
+    - vendor/bin/phpunit --coverage-clover=./tests/report/coverage.clover
+
+after_script:
+    - ./build/coverage_to_scrutinizer.sh
```

Comment:
Ah, yes. I see that now. I had an older copy of dog-statsd that I was looking at. The one-liner makes sense.